portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsSingleSelectionField.testcase

```diff
@@ -32,13 +32,35 @@ definition {
 	}
 
 	@description = "Verify that a Single Selection Field can be created"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test CanBeCreated {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-132189 CanBeCreated pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Single Selection");
+
+		Click(locator1 = "Button#SAVE");
+
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		AppBuilderAdmin.openItem(itemName = "Form 1");
+
+		AssertElementPresent(
+			key_fieldName = "Single Selection",
+			locator1 = "Form#FIELD_LABEL");
```

Comment:
This also works, but it would be better to use the macros that we already have created.
You can use `FormFields.viewRadioLabel` to check if the Single Selection field is rendered, but once it takes two parameters (the field label and field name), you will need to combine it with `FormFields.getFieldName`, which you're going to use to get the fieldname right after creating the field on sidebar.

It would be something like this:

Right after adding the field on sidebar, on line 51
`var fieldName = FormFields.getFieldName();`

And at the end of the test, after entering the form page created:

```
FormFields.viewRadioLabel(
			fieldLabel = "Single Selection",
			fieldName = "${fieldName}");
```

You can find another example [here](https://github.com/liferay/liferay-portal/blob/f620c75a2a63893f3d86f675cd35bc75ef980f2b/portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase#L4406-L4408).